tests/dist/chats.spec.dev.js

```diff
@@ -77,16 +82,16 @@ describe('chat resolvers', function () {
             email = ''.concat(nameForNewUser, '@test.com');
             _context2.next = 8;
             return regeneratorRuntime.awrap(
-            axios.post(URL, {
-              query: '\n              mutation {\n                  signUp(data: {\n                  firstName:"'
-                .concat(nameForNewUser, '",\n                  lastName:"')
-                .concat(nameForNewUser, '"\n                  email: "')
-                .concat(
-                  email,
-                  '"\n                  password:"password"\n                  }) {\n                  user{\n                    _id\n                  }\n                  accessToken\n                  }\n              }\n              '
-                ),
-            })
-          );
+              axios.post(URL, {
+                query: '\n              mutation {\n                  signUp(data: {\n                  firstName:"'
+                  .concat(nameForNewUser, '",\n                  lastName:"')
+                  .concat(nameForNewUser, '"\n                  email: "')
+                  .concat(
+                    email,
+                    '"\n                  password:"password"\n                  }) {\n                  user{\n                    _id\n                  }\n                  accessToken\n                  }\n              }\n              '
```

Comment:
It would be ideal to use a template literal here to improve the readability of this query